pkg/gateway/mirror/usecase/vald.go

```diff
@@ -69,23 +70,23 @@ func New(cfg *config.Data) (r runner.Runner, err error) {
 	// skipcq: CRT-D0001
 	cOpts = append(cOpts, grpc.WithErrGroup(eg))
 
-	client, err := mirror.New(
-		mirror.WithAddrs(cfg.Mirror.Client.Addrs...),
-		mirror.WithClient(grpc.New(cOpts...)),
+	mirrClient, err := client.New(
+		client.WithAddrs(cfg.Mirror.Client.Addrs...),
+		client.WithClient(grpc.New(cOpts...)),
 	)
 	if err != nil {
 		return nil, err
 	}
 
 	gateway, err := service.NewGateway(
 		service.WithErrGroup(eg),
-		service.WithMirrorClient(client),
+		service.WithMirrorClient(mirrClient),
 		service.WithPodName(cfg.Mirror.PodName),
 	)
 	if err != nil {
 		return nil, err
 	}
-	mirror, err := service.NewMirror(
+	mirr, err := service.NewMirror(
```

Comment:
`m` or `ms`(mirror service)

Reply:
Thank you for the suggestion 🙏 
I fixed it!
https://github.com/vdaas/vald/pull/2366/commits/29c8403b5f22af4a6d7f99f3ddee11ac745dcd4f